armi/reactor/reactors.py

```diff
@@ -272,7 +272,7 @@ def __iter__(self):
         --------
         getAssemblies()
         """
-        return iter(sorted(self._children))
+        return iter(self._children)
```

Comment:
the sort would not matter with this sort here since it sorts by something else. i think with John's sorts, this should be good. not sorting itter will also guarantee list accessing and iteration will not be out of sync